lib/storage/metadata/mongoclient/MongoClientInterface.js

```diff
@@ -70,63 +69,57 @@ function generatePHDVersion(versionId) {
  * @param {Object} params - constructor params
  * @param {String} params.host - host for mongo
  * @param {String} params.port - port for mongo
- * @param {String} params.writeConcern - writeConcern for mongo
- * @param {String} params.replicaSet - replicaSet for mongo
```

Comment:
Need to retain `params.replicaSet, params.replicationGroupId ` as it's being passed as a param.